src/config/networks/kovan.ts

```diff
@@ -0,0 +1,47 @@
+import EtherLogo from 'src/config/assets/token_eth.svg'
+import { EnvironmentSettings, ETHEREUM_NETWORK, NetworkConfig } from 'src/config/networks/network.d'
+
+const baseConfig: EnvironmentSettings = {
+  clientGatewayUrl: 'https://safe-client.kovan.staging.gnosisdev.com/v1',
+  txServiceUrl: 'https://safe-transaction.kovan.gnosisdev.com/api/v1',
```

Comment:
This service does not exist either and they are required for the interface to work